### ISSUE TITLE
background_location doesn't work in background.

### ISSUE THREAD

**Issue by TranPhuHung98** [Almoullim/background_location#106]
I followed "How to use" on pub.dev and use this function.

ElevatedButton(
                        onPressed: () async {
                          await BackgroundLocation.setAndroidNotification(
                            title: 'Background service is running',
                            message: 'Background location in progress',
                            icon: '@mipmap/ic_launcher',
                          );
                          //await BackgroundLocation.setAndroidConfiguration(1000);
                          await BackgroundLocation.startLocationService();
                          BackgroundLocation.getLocationUpdates((location) {
                            _bloC.background = _bloC.backgr + 1;
                            print("-------------------------");
                            print(location.latitude.toString());
                            print(location.longitude.toString());
                            print(DateTime.now());
                            print("-------------------------");
                          });
                        },
                        child: Text('Start Location Service')),

But it doesn't work when the app pause (put the app into the background).
I use a real device Android.

**Comment by Ssavas92** [comment]
cihazın izinlerini kontrol et, konum iznini arka planda konum güncellemeleri almadan önce ayarla eğer izin verilmişse uygulamayı başlat.

**Comment by phuhungstdio** [comment]
> cihazın izinlerini kontrol et, konum iznini arka planda konum güncellemeleri almadan önce ayarla eğer izin verilmişse uygulamayı başlat.

Yes, I put permissions in AndroidManifest and the device accepted location permission.
Sometimes it works, but it just work in about 10seconds.

**Comment by alifurkanbudak** [comment]
Maybe check this [answer](https://github.com/Almoullim/background_location/issues/103#issuecomment-873584978)
